repo.go

```diff
@@ -186,6 +186,7 @@ type CommitInfo struct {
 
 type base struct {
 	remote, local string
+	extraArgs     []string
```

Comment:
Since the `extraArgs` is only used by SVN can you put it and the `setExtraArgs` method on the `SVNRepo` type?